lib/elixir/test/elixir/base_test.exs

```diff
@@ -799,13 +799,36 @@ defmodule BaseTest do
         |> Enum.shuffle()
         |> IO.iodata_to_binary()
 
+      allowed_opts =
+        encode
+        |> Function.info()
```

Comment:
Injecting this into the existing identity test preserved the test history. Alternatively, I could break this test into three cases (*16, *32, *64) and extract some helpers. Thoughts?